### PR TITLE
Style Leaderboard

### PR DESCRIPTION
**Pull request by zainsadaqat**
In this pull request, I implemented the following features:
- Use pure CSS
- Google Fonts
- Keep general layout of wireframe
- Improve look and feel of the project

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -3,47 +3,58 @@
...
+          <h2 class="dark-text">Recent scores</h2>
+          <button type="button" class="btn" id="refresh-button">Refresh</button>
         </article>
-        <ul class="leaderboard"></ul>
+        <ul class="leaderboard-table">
+          <li class="list-item">
+            <span class="heading">Name</span>
+            <span class="heading">score</span>
+          </li>
+        </ul>
```
Comment (Dipeshtwis) [inline src/index.html:32]: - [x] I would recommend giving proper margin between `recent score` and `add your score` section to make the UI nice. Please work on that.

### Your Design

Screenshot from 2021-09-03 23-03-42


### Original Design

Screenshot from 2021-09-01 05-05-45

**Review by Dipeshtwis — CHANGES_REQUESTED** [review]
Hi @zainsadaqat, 

:+1: This is good to make it this far, your project is really nice, congratulations on the good work you have done so far.
:+1: README file looks professional.

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


 ## Required Changes ♻️

**_Check the comments under the review_**

- [x] The link is broken in the readme file. Please fix that to make the README professional.

Screenshot from 2021-09-03 23-25-45


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

- You can also include the instruction like `npm i` and `npm start` to install and run the project.

Cheers and Happy coding!👏👏👏
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

**_Dipesh Kumar_**


------

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project(check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by sinansevgi — APPROVED** [review]
# Approved :heavy_check_mark: 

Hi  @zainsadaqat.

Great design :clap:
Your project is complete!  
Congratulations! 🎉

You can merge this pull request and continue with your journey :wave: 


Happy Hacking! :computer: 

-----

You can reply to this conversation if you have any questions or need elaboration. :wave: 
Sinan Sevgi


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/index.html (hunk 1 above)
Comment (zainsadaqat) [reply src/index.html:32]: ## Project has perfect padding and margins.

Screenshot from 2021-09-04 02-03-15

**Comment by zainsadaqat** [comment]
# Thanks, @sinansevgi for approving this pull request!

**Pull request merged**